plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSessionProperties.java

```diff
@@ -189,7 +189,7 @@ public DeltaLakeSessionProperties(
                         false),
                 booleanProperty(
                         PROJECTION_PUSHDOWN_ENABLED,
-                        "Read only required fields from a struct",
+                        "Read only required fields from a row column",
```

Comment:
we can keep it consistent with iceberg ("Read only required fields from a struct")
if change is desired, let's change hive, iceberg and delta to same description (in a follow-up)